retroarch.cfg

```diff
@@ -1,915 +0,0 @@
-## Skeleton config file for RetroArch
```

Comment:
Strange... `retroarch.cfg` is in [.gitignore#L20](https://github.com/libretro/RetroArch/blob/master/.gitignore#L20), but this file is still committed into the repo. While I do agree we should either remove it, or make sure it's not in .gitignore, I feel like this change likely shouldn't happen here.